microplan-init.js

```diff
@@ -24,25 +24,26 @@ var _copyTemplateToPath = function (templateFilePath, destFilePath) {
   })
 }
 program
-  .option('-t, --template <type>', 'specify input template (json|yaml) [yaml]', 'yaml')
+  .option('-t, --template <location>', 'specify input template location (json|yaml) [Default is yaml template]', templateFileLocation.yaml)
   .parse(process.argv)
 
 var initArgs = program.args
 
 if (initArgs.length > 1) {
-  console.error('Filename required')
+  console.error('Filename required. Please specify a valid yaml/json template location as argument.')
   process.exit(1)
 }
 
-if (program.template === 'yaml') {
-  _copyTemplateToPath(templateFileLocation.yaml, initArgs[0])
-} else if (program.template === 'json') {
-  _copyTemplateToPath(templateFileLocation.json, initArgs[0])
-} else {
-  let type = path.extname(program.template)
-  if (type !== '.yml' && type !== '.yaml' && type !== '.json') {
-    console.error('Error : Please select proper template.')
-    process.exit(1)
-  }
-  _copyTemplateToPath(program.template, initArgs[0])
+let fileExt = path.extname(program.template)
+
+if (!fileExt) {
+  console.error('The specified path is a directory. Please specify a valid yaml/json template location.')
+  process.exit(1)
 }
+
+if (fileExt !== '.yml' && fileExt !== '.yaml' && fileExt !== '.json') {
+  console.error('The specified file has extension:', fileExt, '.Please specify a valid yaml/json template location.')
```

Comment:
Since we will remove the `if (!fileExt)` check make this block handle it, lets make error message to 

```js
console.error('File extension not supported. Please use ' + supportedExts.join());
```